### DISCUSSION TITLE
How to install on macOS using Docker compose?

### DISCUSSION THREAD

**Discussion by ruudk** [koush/scrypted#830] [Q&A] [answered]
Is it even possible? I tried the macOS installation method but it downloads the whole internet (100s of brew and pip dependencies) and I don't really like that to have on my machine. So having it scoped in Docker images / container would be ideal.

**Comment by koush** [comment, marked as answer]
You can't due to host network not functioning in docker desktop.

**Comment by koush** [comment]
NVR subscribers have access to a native Mac/windows/linux app though. https://github.com/koush/nvr.scrypted.app/releases

<img width="971" alt="image" src="https://github.com/koush/scrypted/assets/73924/59f449ae-757c-4897-90ab-2cdefb6cbd43">
